typescript/db/src/resolvers/workspace.ts

```diff
@@ -78,10 +84,58 @@ const workspaces = async (
   );
 };
 
+type CreateTutorialDatasetProps = Context & {
+  createdWorkspaceId: string;
+};
+
+const createTutorialDatasetInWorkspace = async ({
```

Comment:
I don't think you actually need this function. Calling `createTutorialDataset` from `createWorkspace` should be suffisient.